gcapi/gcapi.py

```diff
@@ -263,12 +280,27 @@ class ReaderStudiesAPI(APIBase):
     answers = None  # type: ReaderStudyAnswersAPI
     questions = None  # type: ReaderStudyQuestionsAPI
 
+    def detail(self, pk):
+        result = LazyDict(lambda: super().detail(pk))
+        result.ground_truth = GroundTruthAPI(self._client, pk)
+        return result
+
     def ground_truth(self, pk, case_pk):
-        result = self._client(
-            method="GET",
-            path=urljoin(self.base_path, pk + "/ground-truth/" + case_pk),
+        print(
```

Comment:
Don't print warnings! Make this a warning.

Reply:
Yeah, I was wondering about this. Any ideas or preferences on how to solve this. `logging.warning()` ?

Reply:
https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Reply:
Cool thanks! Fixed!